HW1/SoftServeWOstl/main.cpp

```diff
@@ -0,0 +1,74 @@
+#include <iostream>
+#include <string>
+#include <cstdlib>
+#include <algorithm>
+#include <fstream>
+
+
+int main(){
+    char output;
+    std::string roster="ABCDEFGHIJKLMNOPQRSTUVWYZabcdefghijklmnopqrstuvwyz";
+    unsigned int ROSTER_SIZE=roster.size();
```

Comment:
.size() does not return int.

in general, it's best to use unsigned int's for bits manipulations